lms/validation/_oauth.py

```diff
@@ -0,0 +1,9 @@
+"""Validation for OAuth views."""
+from webargs import fields
+
+
+#: Arguments for the canvas_oauth_callback() view.
+CANVAS_OAUTH_CALLBACK_ARGS = {
+    "code": fields.Str(required=True),
+    "state": fields.Str(required=True),
+}
```

Comment:
This is the actual validation schema. You can write these as full on [marshmallow](https://marshmallow.readthedocs.io/) schema classes (marshmallow is the validation lib that webargs wraps), but webargs also provides this easy dict format.

Note that just because code and state are present and are strings, doesn't mean they're valid. They need to match values in the DB. I think it would be possible to make the schema check that too. But I haven't done so yet.

The idea is that this `_oauth.py` module will grow to contain all the OAuth-related schemas, and more `_foo.py` modules will be added to contain more schemas.